pkg/cvo/metrics.go

```diff
@@ -432,6 +438,21 @@ func (m *operatorMetrics) Collect(ch chan<- prometheus.Metric) {
 			ch <- g
 		}
 
+		enabledCapabilities := make(map[configv1.ClusterVersionCapability]struct{}, len(cv.Status.Capabilities.EnabledCapabilities))
+		for _, capability := range cv.Status.Capabilities.EnabledCapabilities {
+			g := m.capability.WithLabelValues(string(capability))
+			g.Set(float64(1))
+			ch <- g
+			enabledCapabilities[capability] = struct{}{}
+		}
+		for _, capability := range cv.Status.Capabilities.KnownCapabilities {
```

Comment:
I am not clear on why we need to send known capabilities.

Reply:
Two use cases:

* conditional updates using local PromQL can distinguish between "yeah, that cap is not enabled here" (value 0) and "err, something's broken with scraping this metrics" (no match)
* Telemetry can gauge the install-fraction for a capability (`countOfClustersWithCapEnabled / countOfClustersWithCapKnown`).  with `count by (name) (cluster_version_capability == 1) / count by (name) (cluster_version_capability)`.  If `cluster_version_capability` did not include known-but-not-enabled caps, you'd need some added filtering to get at `countOfClustersWithCapKnown`.